features/core/src/main/java/org/apache/karaf/features/internal/service/FeaturesServiceImpl.java

```diff
@@ -122,13 +101,6 @@
      */
     private final Bundle bundle;
     private final BundleContext bundleContext;
-
-    /**
-     * The system bundle context.
-     * For all bundles related operations, we use the system bundle context
-     * to allow this bundle to be stopped and still allow the deployment to
-     * take place.
-     */
     private final BundleContext systemBundleContext;
```

Comment:
I moved the comment to Regions as I think the functionality refered by the comment is also moved. The remaining use of systemBundleContext is in getDeploymentState and I had the impression the comment does not match the usage there.. btw. as only one method uses systemBundleContext we can maybe move it out of FeaturesServiceImpl in a later refactoring.